components/dialog/demo/dialog-confirm.html

```diff
@@ -22,7 +22,7 @@ <h2>Confirm Dialog</h2>
 			<d2l-demo-snippet>
 				<template>
 					<d2l-button id="openConfirm">Show Confirm</d2l-button>
-					<d2l-dialog-confirm id="confirm" text="Are you sure you want more cookies?">
+					<d2l-dialog-confirm id="confirm" title-text="Confirm Title" text="Are you sure you want more cookies?">
```

Comment:
This had been removed in a previous PR but there's another example on the page that's specifically for this case so I re-added this.